lib/fs/fakefs.go

```diff
@@ -429,12 +467,20 @@ func (fs *fakefs) Rename(oldname, newname string) error {
 	fs.mut.Lock()
 	defer fs.mut.Unlock()
 
+	oldBase := filepath.Base(oldname)
+	newBase := filepath.Base(newname)
```

Comment:
Maybe `oldKey` and `newKey` to make their purpose even more explicit.

Reply:
Fair point, will adjust.